msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+### YamlMime:Tutorial
+title: Graph activity feed broadcast
```

Comment:
Can we change the title of the doc to 'Send activity feed notification'. As per my understanding, you can broadcast and notify activity feed using Microsoft Graph API.

Reply:
Comments incorporated.